examples/core_api/endpoints/time_now_endpoint.rb

```diff
@@ -27,9 +27,7 @@ class TimeNowEndpoint < Apia::Endpoint
       field :my_polymorph, type: Objects::MonthPolymorph do
         description "A polymorphic field!"
       end
-      field :my_partial_polymorph, type: Objects::MonthPolymorph, include: "number", null: true do
-        description "Partial polymorph"
-      end
+      field :my_partial_polymorph, type: Objects::MonthPolymorph, include: "number", null: true
```

Comment:
I removed the description here to test that we still set nullable correctly even when there is no description for the field.